### PR TITLE
Move info from DevTools > "Accessibility-testing features" to A11y > "Resources for accessibility testing"

### PR DESCRIPTION
**Pull request by captainbrosset**
Rendered articles for review:

* **DevTools** TOC bucket > **Accessibility-testing features** article
   * https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/accessibility/reference?branch=pr-en-us-3045
   * [Public preview](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/devtools-a11y/microsoft-edge/devtools-guide-chromium/accessibility/reference.md)
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/accessibility/reference)

* **Accessibility** TOC bucket > **Resources for accessibility testing** article > sections: **Accessibility testing features in DevTools**; **Additional manual accessibility testing**; **Use testers who have different accessibility needs**; **Use Accessibility Insights**
   * https://review.learn.microsoft.com/microsoft-edge/accessibility/test?branch=pr-en-us-3045#accessibility-testing-in-devtools
   * [Public preview](https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/devtools-a11y/microsoft-edge/accessibility/test.md#accessibility-testing-in-devtools)
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/accessibility/test#accessibility-testing-in-devtools)

* toc.yml
   * In **Accessibility** TOC bucket, made link text match destination titles, to fix broken nav due to contradictory titles. - m.h.
   * Removed empty `displayName:` lines, to discourage contradictory TOC titles. - m.h.

This PR moves the content that's currently at the end of the DevTools **Accessibility-testing features** ("feature-Reference") article to the general accessibility-testing docs instead (the **Resources for accessibility testing** article).  The more-general accessibility-testing article is a much better place for this content.

Rationale: 
* The goal & purpose of the DevTools accessibility-testing reference article (**Accessibility-testing features**) is to answer HOW/WHERE (in UI) questions about the accessibility-testing features of DevTools.  
* The article shouldn't have to concern itself with answering WHY questions (convincing people to test), or providing general ideas for automated and manual testing.

AB#48058322

### CODE REVIEW

File: microsoft-edge/accessibility/test.md [hunk 1]
```diff
@@ -18,7 +18,41 @@ Use the following tools and testing procedures to evaluate your website for acce
 <!-- ====================================================================== -->
 ## Accessibility testing in DevTools
```
Comment (mikehoffms) [inline microsoft-edge/accessibility/test.md:20]: unable to navigate.  might need index entry in toc
image

Comment (mikehoffms) [reply microsoft-edge/accessibility/test.md:20]: made Accessibility TOC entries match top-of-article titles

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/accessibility/reference.md [hunk 2]
```diff
@@ -78,54 +78,10 @@ This article lists typical accessibility aspects to test for webpages, and the c
...
-
-Automated tools can't find all the problems in a product, because many of the accessibility barriers show up only during interactive use.  None of these features can replace a proper round of testing with people that use assistive technologies and following a plan to check for all the required tests.
-
-
-#### Use Accessibility Insights
-
-You can also use the [Assessments](https://accessibilityinsights.io/docs/en/web/getstarted/assessment/) feature of [Accessibility Insights](https://accessibilityinsights.io).
-
-
-#### Use webhint in Visual Studio Code
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/accessibility/reference.md:120]: @captainbrosset - did you deliberately remove section "Use webhint in Visual Studio Code"?

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/accessibility/reference.md:120]: Yes, webhint is available through our DevTools VSCode extension.

**Pull request merged**